python/src/cucumber_messages/json_converter.py

```diff
@@ -0,0 +1,321 @@
+import json
+import re
+import sys
+import types
+import typing
+from dataclasses import MISSING, Field, fields, is_dataclass
+from datetime import date, datetime
+from enum import Enum
```

Comment:
Considering we are targetting python >=3.9, I was surprised that we are using typing imports for things like Dict, List, Union, etc. instead of the built-ins?